ig/input/fsh/nbs-orc-segment.fsh

```diff
@@ -2,15 +2,20 @@ Logical: ORCSegment
 Id: segment-orc-logical-model
 Title: "ORC Segment"
 Description: "The ORC Segment"
-* orderControl 1..1 SU string "ORC-1 Order Control"
-* ORCPlacerOrderNumber 1..1 SU string "ORC-2 Placer Order Number"
-* fillerOrderNumber 1..1 SU string "ORC-3 Filler Order Number"
-* placerGroupNumber 1..1 SU string "ORC-4 Placer Group Number"
-* dateTimeOfTransaction 1..1 SU string "ORC-9 Date/Time of Transaction"
-* orderingProvider 1..1 SU string "ORC-12 Ordering Provider"
-* orderingFacilityName 1..1 SU string "ORC-11 prdering Facility Name"
-// other optional fields
-
+* orderControl 1..1 SU id "ORC-1 Order Control"
+* placerOrderNumber 1..1 SU Identifier "ORC-2 Placer Order Number"
+* placerOrderNumberEntityIdentifier 1..1 SU string "ORC-2.1 Placer Order Number Entity Identifier"
+* placerOrderNumberNamespaceID 1..1 SU code "ORC-2.2 Placer Order Number Namespace ID"
+* placerOrderNumberUniversalID 1..1 SU string "ORC-2.3 Placer Order Number Universal ID"
+* placerOrderNumberUniversalIDType 1..1 SU id "ORC-2.4 Placer Order Number Universal ID Type"
```

Comment:
I think the general gist is that when a field (like `ORC-2.2`) is controlled but is completely [user defined](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0363), then it is a `string` because it's fairly open ended still.  On the other hand, `ORC-2.4` is controlled by a very distinct set of controlled options ([table 0301](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0301) specifically).

```suggestion
* placerOrderNumberNamespaceID 1..1 SU string "ORC-2.2 Placer Order Number Namespace ID"
* placerOrderNumberUniversalID 1..1 SU string "ORC-2.3 Placer Order Number Universal ID"
* placerOrderNumberUniversalIDType 1..1 SU code "ORC-2.4 Placer Order Number Universal ID Type"
```